server/src/main/java/org/apache/kafka/server/AssignmentsManager.java

```diff
@@ -17,6 +17,9 @@
 
 package org.apache.kafka.server;
 
+import com.yammer.metrics.core.Gauge;
```

Comment:
`:server:spotlessApply` isn't happy with the import order

Reply:
yes, I fixed this